near-sdk/src/store/mod.rs

```diff
@@ -1,2 +1,4 @@
 mod lazy;
 pub use lazy::Lazy;
+mod vec;
+pub use vec::Vector;
```

Comment:
I think we need to make the `vec` module itself public, so that the cilents can name `Iter` strcuts.